content/en/platform/corda/5.0-alpha/deploying/deployment-options/network-types.md

```diff
@@ -39,7 +39,7 @@ To run a dynamic network, you must complete the following high-level steps:
 3. Package the MGM `GroupPolicy.json` file into an MGM CPI.
 4. Upload the CPI to your cluster.
 5. Create a virtual node in your cluster for the MGM.
-6. Assign required HSMs for the MGM.
+6. Assign required hardware security models (HSMs) for the MGM.
```

Comment:
Should be:
`Hardware Security Modules`
(capital letter per word and Modules instead of Models)